pkg/zval/encoding.go

```diff
@@ -46,6 +46,12 @@ func (e Encoding) Build(b []byte) ([]byte, error) {
 			return nil, err
 		}
 		if container {
+			if v == nil {
+				b = append(b, '(')
+				b = append(b, '*')
+				b = append(b, ')')
+				continue
+			}
```

Comment:
Given that we've dropped the * convention, should this still be here?

Reply:
Yes because we're showing the difference between container-unset and value-unset which exists in the underlying zval but would be lost if we stringified them both the same way.

Reply:
This string format doesn't conform to any sort of spec.

Reply:
In that case, you can shorten this to `b = append(b, "(*)"...)` in the future.